android/src/main/java/com/revenuecat/purchases/react/RNPurchasesConverters.kt

```diff
@@ -62,6 +62,7 @@ internal object RNPurchasesConverters {
                 null -> writableArray.pushNull()
                 is Boolean -> writableArray.pushBoolean(item)
                 is Int -> writableArray.pushInt(item)
+                is Long -> writableArray.pushDouble(item.toDouble())
```

Comment:
Double isn't really a Long, but I couldn't find an alternative on a quick search 😞. I guess we can do this, as long as we handle it well in the react native side.

Reply:
Yeah, I didn't love this 😬 Double was the only thing that could fit a long and it works soooooo... 🤷‍♂️